docs/include_action-types/document-template.rst

```diff
@@ -0,0 +1,38 @@
+.. _document-template:
+
+Document template
+-----------------
+
+If you are modelling a workflow, there is often the need to collect the information input entered by users of the workflow tasks.
+These data are added dynamically to a document, which is then used as a basis for a later audit, for example for further decisions.
+Because data are collected in a Word file, the content can be modified subsequently.
+To do this, add the action type 'Document template' in your workflow.
+With this action, a task is assigned to users including the request to specify the necessary information in a custom task form.
+The information retrieved from this task form is inserted by Workflow Accelerator in the uploaded template.
+
+Suppose your company has set up a workflow for contracts or quotations, in which several departments must define relevant data through the assignment of workflow tasks.
+The contract or quotation is created based on a standard template, so that required information from the workflow are assigned accordingly in an output form.
```

Comment:
_ information_ *is* assigned *to* an output form.
Information can't be singular. But perhaps it's better to continue to use the term _data_, which can be plural (depending on the opinion, because it's derived from the Latin plural of _datum_)